modules/mobile/app/models/mobile/v2/adapters/appointments.rb

```diff
@@ -0,0 +1,328 @@
+            facility_id: facility_id,
+            sta6aid: sta6aid,
+            healthcare_provider: healthcare_provider(appointment_hash[:practitioners]),
+            healthcare_service: healthcare_service(appointment_hash, type),
+            location: location(type, appointment_hash),
+            minutes_duration: minutes_duration(appointment_hash[:minutes_duration], type),
+            phone_only: appointment_hash[:kind] == PHONE_KIND,
+            start_date_local: start_date_local,
+            start_date_utc: start_date_utc,
+            status: status,
+            status_detail: cancellation_reason(appointment_hash[:cancelation_reason]),
+            time_zone: time_zone,
+            vetext_id: vetext_id(appointment_hash, start_date_local),
+            reason: appointment_hash.dig(:reason_code, :coding, 0, :code),
+            is_covid_vaccine: appointment_hash[:service_type] == COVID_SERVICE,
+            is_pending: status == STATUSES[:proposed],
+            proposed_times: proposed_times(appointment_hash[:requested_periods]),
+            type_of_care: type_of_care(appointment_hash[:service_type], type),
+            patient_phone_number: contact(appointment_hash.dig(:contact, :telecom), CONTACT_TYPE[:phone]),
+            patient_email: contact(appointment_hash.dig(:contact, :telecom), CONTACT_TYPE[:email]),
+            best_time_to_call: appointment_hash[:preferred_times_for_phone_call],
+            friendly_location_name: appointment_hash.dig(:extension, :cc_location, :practice_name)
+          }
+
+          Rails.logger.info('metric.mobile.appointment.type', type: type)
+
+          Mobile::V0::Appointment.new(adapted_hash)
+        end
+
+        def cancel_id(appointment_hash)
+          return nil unless appointment_hash[:cancellable]
+
+          appointment_hash[:id]
+        end
+
+        def type_of_care(service_type, type)
+          va?(type) ? nil : service_type
+        end
+
+        # rubocop:enable Metrics/MethodLength
+        def cancellation_reason(cancellation_reason)
+          return nil unless cancellation_reason
+
+          cancel_code = cancellation_reason.dig(:coding, 0, :code)
+          CANCELLATION_REASON[cancel_code.to_sym]
+        end
+
+        def contact(telecom, type)
+          return nil if telecom.blank?
+
+          telecom.select { |contact| contact[:type] == type }&.dig(0, :value)
+        end
+
+        def proposed_times(requested_periods)
+          return nil if requested_periods.nil?
+
+          proposed_times = []
+          requested_periods.each do |period|
+            start_date = DateTime.parse(period[:start])
+            proposed_times << {
+              date: start_date.strftime('%m/%d/%Y'),
+              time: start_date.hour.zero? ? 'AM' : 'PM'
+            }
+          end
+
+          proposed_times
+        end
+
+        def vetext_id(appointment_hash, start_date_local)
+          "#{appointment_hash[:clinic]};#{start_date_local.strftime('%Y%m%d.%H%S%M')}"
```

Comment:
what is this used for? is it still relevant?

Reply:
I'm not sure either. We could probably check the mobile app code for references to it.

Reply:
@jmarchi Do you know if the response field `vetext_id` is used by the FE? As far as I can tell we don't use it on the BE we just set it.

Reply:
The only place I see it being used is our mock data so I don't believe so

Reply:
sounds good. I will set the field as nil to avoid any unforeseen issues with it.